### PR TITLE
Fix unable to display individual-level project

### LINKED ISSUE #26189: Unable to add issues to user level project
### Description

I created a user project from `https://try.gitea.io/<username>/-/projects -> New Project`. However I don't see how to add issues to this project. Tried few things.

1. From projects page `https://try.gitea.io/<username>/-/projects/<project_id>`, don't see any option to add issue 
2. Created an issue in of the repository. Create page `https://try.gitea.io/<username>/<repo_name>/issues/new has a 'Projects' setting on the setting. But don't see the user project in this list. 
3. Project does not show up in issue listing page or the issue page
4. Tried assigning the project to the user and still don't see any option to add to a user project.

Is there any way other way to use the user project ?

Link: https://try.gitea.io/balki/-/projects/3724

### Gitea Version

1.21.0+dev-408-geaea530d6

### Can you reproduce the bug on the Gitea demo site?

Yes

### Log Gist

_No response_

### Screenshots

_No response_

### Git Version

_No response_

### Operating System

_No response_

### How are you running Gitea?

https://try.gitea.io/

### Database

None

### PR DESCRIPTION
**Pull request by CaiCandong**
As title

Before:
image

After:
image

fix #26189

### CODE REVIEW

**Review by delvh — APPROVED** [review]

**Comment by delvh** [comment]
Alternatively, we can also combine `projects2` and `projects3` by removing the `Type` condition.
That's probably a better idea anyways, as there are more types than that.

**Comment by CaiCandong** [comment]
> That's probably a better idea anyways, as there are more types than that.

It may be that my previous code wasn't clear enough, leading you to think it should be changed to this title:`Show open user projects too when displaying open projects`.
I'm actually trying to fix https://github.com/go-gitea/gitea/issues/26189, The title of this should be：`Fix unable to display individual-level project`

File: routers/web/repo/issue.go [hunk 1]
```diff
@@ -518,6 +518,12 @@ func RetrieveRepoMilestonesAndAssignees(ctx *context.Context, repo *repo_model.R
 }
 
 func retrieveProjects(ctx *context.Context, repo *repo_model.Repository) {
+	// Distinguish whether the owner of the repository
+	// is an individual or an organization
+	repoOwnerType := project_model.TypeIndividual
+	if repo.Owner.IsOrganization() {
```
Comment (lunny) [inline routers/web/repo/issue.go:524]: Can you ensure repo.Owner has been loaded from database? Maybe invoke `repo.LoadOwner` first.

Comment (CaiCandong) [reply routers/web/repo/issue.go:524]: I checked the invoke of `retrieveProjects`, and in total the following three. I  confirmed owner has been loaded from the database. 

https://github.com/go-gitea/gitea/blob/55532061c83d38d33ef48bdc5eeac0f652844e8a/routers/web/repo/issue.go#L433
https://github.com/go-gitea/gitea/blob/55532061c83d38d33ef48bdc5eeac0f652844e8a/routers/web/repo/issue.go#L761
https://github.com/go-gitea/gitea/blob/55532061c83d38d33ef48bdc5eeac0f652844e8a/routers/web/repo/issue.go#L1262

**Review by puni9869 — APPROVED** [review]

**Review by lunny — APPROVED** [review]

**Comment by CaiCandong** [comment]
The purpose of this PR is to fix Bug #26189. I have verified that this bug also exists in previous versions, so the PR should be backported to those earlier versions.

**Pull request merged**